### DISCUSSION TITLE
Viewing `rds` files?

### DISCUSSION THREAD

**Discussion by rkrug** [posit-dev/positron#3891] [Q&A] [answered]
I might be missing it - but is there an easy way of viewing `rds` files? Like the viewers for e.g. parquet (e.g. parquet-viewer extension) to simply view the contents of an `rds` file?

**Comment by jthomasmock** [comment, marked as answer]
Thanks @rkrug -- the Data Explorer is currently focused on in-memory data for R/Python.

We may explore on-disk views, likely starting with cross-language formats like parquet, CSV, and perhaps arrow. See more at #2187

**Comment by rkrug** [comment]
Thanks @jthomasmock . That is exactly what I was looking for.

I hope in `arrow` you also include `parquet`?

Looking forward to that - sounds great! Let me know if there is a something I could test..

**Reply by jthomasmock** [reply]
Great! Nothing to report or test yet, but you might subscribe to: https://github.com/posit-dev/positron/issues/2187 to keep up to date or ask questions/share interest.

**Comment by rkrug** [comment]
Done. Thanks.
